internal/app/config.go

```diff
@@ -122,3 +129,41 @@ func (cfg *Configuration) validate() error {
 
 	return nil
 }
+
+// setConfigDefaults sets all values in the struct to empty. This is to get around a weird quirk of viper.
```

Comment:
This might be a cleaner solution ? https://github.com/metal-toolbox/flasher/blob/ca39a7008988a827b4b3fc676c67ce479a2f87e1/internal/app/config.go#L152 or the one linked in the comment there https://github.com/spf13/viper/pull/1429